README.md

```diff
@@ -82,6 +82,9 @@ const combobo = new Combobo();
     ```
 * `filter` (_String|Function_): A filter-type string (`'contains'`, `'starts-with'`, or `'equals'`) or a function that returns a array of filtered options.
   * Defaults to `'contains'`
+* `onPageFilter` (_Boolean_): To enable / disable filterng options on front end. If the developer wants to filter options from the server, then it should be false
```

Comment:
This property name doesn't do a great job explaining its purpose (to me, `onPageFilter` seems more like a hook that is called when something happens since it starts with "on"). How about something like `autoFilter` (defaulting to `true`)?